### PR TITLE
Update nativescript-vue.json

### PR DESCRIPTION
**Pull request by rigor789**
# Pull request motivation(s)
Added language variable, changed selectors to the ones we have added to the site structure

### CODE REVIEW

File: configs/nativescript-vue.json [hunk 1]
```diff
@@ -17,19 +18,20 @@
   "stop_urls": [],
   "selectors": {
     "lvl0": {
-      "selector": "//li[contains(@class, 'level-0')]//a[contains(@class, 'docSearch-active')][1]/preceding::span[1]",
+      "selector": "selector": "//li[contains(@class, 'level-0')]//a[contains(@class, 'docSearch-active')][1]/preceding::span[1]",
```
Comment (s-pace) [inline configs/nativescript-vue.json]: Mistake here

**Comment by s-pace** [comment]
✅ Deploy

File: configs/nativescript-vue.json (hunk 1 above)
Comment (rigor789) [reply configs/nativescript-vue.json]: Ah yes thanks! Tinkered with it and left it like that accidentally!

**Comment by s-pace** [comment]
- Copy the following CSS/JS snippets and add them to your page

```html
<!-- at the end of the HEAD -->
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" />

<!-- at the end of the BODY -->
<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
<script type="text/javascript"> docsearch({
  apiKey: '31f842779e5b33726ec73f8f973b2202',
  indexName: 'nativescript-vue',
  inputSelector: '### REPLACE ME ####',
  algoliaOptions: { 'facetFilters': ["lang:$LANG"] },
  debug: false // Set debug to true if you want to inspect the dropdown
});
</script>
```

- Add a search input in your page if you don't have any yet. Then update the inputSelector value in JS snippet to a CSS selector that targets your search input field.

- Replace $LANG with the lang you want to search on.
   The list of possible lang is hardcoded in the config.
   So as of today you have: en, ko

  For example if you want to refine the search to the lang "en" just specify: 'facetFilters': ["lang:en"]

- Optionally customize the look and feel by following the [DocSearch documentation](https://community.algolia.com/docsearch/documentation/)
- You can also check your configuration in [our github repo](https://github.com/algolia/docsearch-configs/blob/master/configs/nativescript-vue.json).

**Pull request merged**